features/eolearn/tests/test_feature_manipulation.py

```diff
@@ -232,14 +233,38 @@ def test_linear_function_task():
 @pytest.mark.filterwarnings("ignore::RuntimeWarning")
 def test_spatial_resize_task(example_eopatch):
```

Comment:
the test is getting kinda long, would it be possible to parametrize it?
So to figure out how to get an eopatch of twice the size and half the size (for each of the input modes) and then write all the possible parameters expressing that. The test for faulty parameters is then a separate one. Also test that strings are correctly parsed to enums (so a test case should use `"resolution"`) if we are keen on keeping that shortcut for users.

Reply:
done